docs/content/docs/deployment/azure-frs.md

```diff
@@ -1,5 +1,94 @@
+
+```typescript
+const config: FrsConnectionConfig = {
+    tenantId: "myFrsTenantId",
+    // IMPORTANT: this token provider is suitable for testing ONLY. It is NOT secure.
+    tokenProvider: new InsecureTokenProvider("myFrsTenantKey", { id: "UserId", name: "Test User" }),
+    orderer: "https://myFrsOrdererUrl",
+    storage: "https://myFrsStorageUrl",
+}
+
+const client = new FrsClient(config);
+```
+
+Now that we have an instance of our client, we can start using it to create new or load existing Fluid containers!
+
+# Managing Containers
+
+The `FrsClient` provides two functions on its API to create and get containers respectively. They both take in two parameters:
+- A container config that defines the ID of the container and an optional entrypoint for logging
+- A container schema that defines the DDSes and data objects that this container will hold
+
+```typescript
+const schema = {
+    name: "my-container",
+    initialObjects: {
+        /* ... */
+    },
+    dynamicObjectTypes: [ /*...*/ ],
+}
+const frsClient = new FrsClient(config);
+await frsClient.createContainer({ id: "_unique-id_" }, schema);
+const { fluidContainer, containerServices } = await frsClient.getContainer({ id: "_unique-id_" }, schema);
+```
+
+The ID that is being passed in to the container config can be thought of as the "filename" for the item holding the data for this container on the FRS backend. Any client that wants to join the same collaborative session just needs to call `getContainer` with the same container ID.
+
+For the further information on how to start recording logs being emitted by Fluid, please see [Telemetry](../testing/telemetry.md)
+
+The container being fetched back will hold the `initialObjects` as defined in the container schema. Please read [Data modeling](../build/data-modeling.md) to see how to establish the schema and use the `fluidContainer` object.
+
+# Getting Audience Details
+
+Both calls to `createContainer` and `getContainer` return an `FrsResources` object that holds the Fluid container that we were discussing above as well as a `containerServices` object. Whereas the container itself will always stay the same regardless of which service it is being connected, the `containerServices` hold values that are specific to the FRS service. Within this object, we will find an `audience` value that can be used to manage the roster of users that are currently collaborating in the container.
```

Comment:
```suggestion
Both calls to `createContainer` and `getContainer` return an `FrsResources` object that holds the Fluid container described above as well as a `containerServices` object.

The Fluid container is service-agnostic. If you were to retrieve a Fluid container from a different Fluid service -- not FRS -- the code you wrote against that container using FRS is portable between services.

The `containerServices` object contains data that is specific to the FRS service. This object contains an `audience` value that can be used to manage the roster of users that are currently collaborating in the container.
```

The container service-agnostic thing is _really_ tricky to communicate! I don't love my changes. Maybe just using the class name instead would help?

Reply:
```suggestion
Calls to `createContainer` and `getContainer` return an `FrsResources` object that contains a `FluidContainer` -- described above -- and a `containerServices` object.

The `containerServices` object contains data that is specific to the Azure Fluid Relay service. This object contains an `audience` value that can be used to manage the roster of users that are currently connected to the container.
```

Reply:
Thank you for the suggestions here @tylerbutler and @skylerjokiel! It's definitely sounding a lot clearer with these recommendations and I think conveying the "service-agnostic" aspect is going to be key for ppl to understand so they know they can keep moving their container between services. Just made a small addition to specify that this means that they don't have to change any of their interactions with DDSes and data objects that are within their container

```suggestion
Calls to `createContainer` and `getContainer` return an `FrsResources` object that contains a `FluidContainer` -- described above -- and a `containerServices` object.

The `FluidContainer` contains the Fluid data model and is service-agnostic. Any code you write against this container object returned by the `FrsClient` is reusable with the client for another service. An example of this is if you prototyped your scenario using `TinyliciousClient`, then all of your code interacting with the Fluid DDSes and data objects within the container can be reused when moving to using `FrsClient`.

The `containerServices` object contains data that is specific to the Azure Fluid Relay service. This object contains an `audience` value that can be used to manage the roster of users that are currently connected to the container.
```